meeting-notes/2019-06-20-notes.md

```diff
@@ -0,0 +1,172 @@
+
+5. Open Issues
+How should we handle this todo list? maybe a space in our discourse forum that gets referenced in our checkins?? This gets into task management which i think we can do in github but i have always used asana :) 
+Open Issues 
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/17
+    - [ ] sign up here
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/27
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/28
+
+4. Plan to discuss gitter for pyOpenSci in future meeting
+
+5. Roles reminder (we can skip this if there's not time today)
+
+## Role Definition Revisited (please feel free to edit this or leave comments!!)
+
+Starting with the [rOpenSci structure](https://ropensci.org/about/) but we may want to adjust accordingly. And ofcourse the time commitment can really be flexible but just trying to give folks a sense of time that might be associated with each role. 
+
+## Leadership (3-5): 
+* Responsible for making final decisions about the direction of POS
+* POS
+* Communication: weekly / bi weekly, slack, discuss private forum (Chris H can help us set this up i think??, emails)
+    * Gitter https://gitter.im/pyOpenSci/
+
+## Community, Social Media, Outreach, Website (?future or someone doing part):
+Ropensci has an outreach person. I wonder if we could start with anyone who had
+just a bit of time to help us start to build an online presence. Potential for a fall intern... 
+(Neil: we could also apply to https://www.outreachy.org/)
+* website
+* twitter
+* spreading the word (easy peasy just tell people about us)
+
+## Editors in Chief (2-3?):
+* What about defining this initially as editors ?
+* All editors will be pinged when a new package is submitted. Then based on workload, one will be assigned to it as an editor. 
+    * Luiz: I would like to be in this group, to eventually bootstrap the bioinformatics area =]
+    * _Depending on load, this might be an area Carson could contribute as well_
+    * Martin
+
+##  Pool of Associate Editors (8+)
+A group of people who are willing to fill the editor role for a single package for a shorter duration of  time throughout the year. Associate editors will be pinged by the core editor group to assess their time availability and expertise relative to serving in an editor capacity for a particular capacity. Their time can be more limited than the time required of the editors in chief. 
+* General contribution load: 2-4 packages a year?? (_just throwing out a number here_)
+
+NOTES: JOSS model: Joss has a small number (4) of editors in chief. each week one is on call and responsible for everything. has a larger pool of associated editors. this might work better if there is a larger community who want to jump in here and there. 
+
+## Advisory (4*):
+### Responsibilities:
+* Provide feedback & guidance regarding the direction of POS
+* Spread the word about POS
+* Communication: every other month / quarterly? meetings??
+---
+* Carson  (external/industry advisor)
+* Neil
```

Comment:
right now this is probably the role that I can most-confidently commit to

Reply:
oh @choldgraf feel free to add your name here. we can create a running list of roles somewhere more visible as well if we want!